content/attacks/posts/2016-06-17-DAO.md

```diff
@@ -0,0 +1,35 @@
+---
+date: 2016-06-17
+custodians: The DAO, Ethereum
+categories: Contract Flaw
+title: DAO falls prey to smart contract exploit, resulting in $60 million Ether theft
+---
+
+## Summary
+
+On June 17, 2016, an exploit in the DAO (Decentralized Autonomous Organization), a venture capital fund on the Ethereum blockchain, allowed an attacker to drain approximately $60 million worth of Ether into a separate entity referred to as the "Child DAO."
+
+## Attackers 
+
+The perpetrator behind the DAO exploit remains undisclosed. However, they manipulated a recursive calling vulnerability in the DAO's smart contract, which permitted repeated Ether withdrawals before the smart contract could update the balance.
+
+## Losses
+
+The DAO exploit led to an estimated $60 million in Ether being transferred to the Child DAO controlled by the attacker.
+
+## Timeline
```

Comment:
the timeline section needs to be significantly expanded. there are plenty of detailed descriptions of the events out there